pkg/sql/plan/function/builtin/unary/load_file.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2022 Matrix Origin
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package unary
+
+import (
+	"context"
+	"errors"
+	"io"
+	"strings"
+
+	"github.com/matrixorigin/matrixone/pkg/container/types"
+	"github.com/matrixorigin/matrixone/pkg/container/vector"
+	"github.com/matrixorigin/matrixone/pkg/fileservice"
+	"github.com/matrixorigin/matrixone/pkg/vm/process"
+)
+
+func LoadFile(vectors []*vector.Vector, proc *process.Process) (*vector.Vector, error) {
+	inputVector := vectors[0]
+	resultType := types.New(types.T_varchar, 0, 0, 0)
+	vec := vector.New(resultType)
+	const blobsize = 65536 // 2^16-1
+	Filepath := vector.GetStrColumn(inputVector).GetString(0)
+	r, err := ReadFromLocalFile(Filepath)
+	if err != nil {
+		return nil, err
+	}
+	ctx, err := io.ReadAll(r)
+	defer r.Close()
+	if len(ctx) > blobsize {
+		return nil, errors.New("Data too long for blob")
+	}
+	if err := vec.Append(ctx, proc.GetMheap()); err != nil {
+		return nil, err
+	}
+	return vec, nil
+}
+
+func ReadFromLocalFile(Filepath string) (io.ReadCloser, error) {
```

Comment:
Why do you have to read from local?   Can we make Filepath an URL and let fileservice read from local or s3?